src/sql/_testing.py

```diff
@@ -316,7 +316,7 @@ def mssql(is_bypass_init=False):
                 "MSSQL_SA_PASSWORD": db_config["password"],
                 "ACCEPT_EULA": "Y",
             },
-            ready_test=lambda: database_ready(database="MSSQL"),
+            ready_test=lambda: database_ready(database="MSSQL", timeout=60),
```

Comment:
I think we can add this timeout for every database not just "MSSQL"

Reply:
fix in [1ae8673](https://github.com/ploomber/jupysql/pull/493/commits/1ae867365472eedb110e4bbf9ee42f7f5ba53a97)